lib/tarantool_server.py

```diff
@@ -825,10 +828,13 @@ def start(self, silent=True, wait=True, wait_load=True, rais=True, args=[],
 
         # redirect stdout from tarantoolctl and tarantool
         os.putenv("TEST_WORKDIR", self.vardir)
+        saved_cwd = os.getcwd()
+        os.putenv("PWD", self.vardir)
```

Comment:
This hell really deserves its own description, so I'll extract this change into its own commit. Hope you don't mind.

Reply:
Extracted into its own commit.